src/main/java/com/unbxd/SchemaManager/Dao/RedisDao.java

```diff
@@ -0,0 +1,59 @@
+package com.unbxd.SchemaManager.Dao;
+
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.unbxd.SchemaManager.Models.Field;
+import com.unbxd.SchemaManager.Models.SiteSchema;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Component;
+import redis.clients.jedis.Jedis;
+
+import java.io.IOException;
+
+@Component
+@Qualifier("Redis")
+public class RedisDao implements Dao {
+
+    Jedis jedis = new Jedis("localhost");
+    ObjectMapper mapper = new ObjectMapper();
```

Comment:
same, 
run a grep of new in your source code
every new call should be in providers